packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -375,15 +375,16 @@ apiDescribe('Database transactions', persistence => {
     return integrationHelpers.withTestDb(persistence, db => {
       const doc1 = db.collection('counters').doc();
       const doc2 = db.collection('counters').doc();
-      let tries = 0;
+      // let tries = 0;
       return (
         doc1
           .set({
             count: 15
           })
           .then(() => {
             return db.runTransaction(transaction => {
-              tries++;
+              // ++tries;
```

Comment:
rm? And if you do so, I think you could optionally simplify line 385-390 slightly:

```
return db.runTransaction(transaction -> transaction.get(doc1).then(...).then(...))
```

Reply:
See above.